packages/cvxpy-base/test_cvxpy.py

```diff
@@ -0,0 +1,47 @@
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide(packages=["cvxpy"])
```

Comment:
The package name here should follow the name in the meta.yaml. So this needs to be `cvxpy-base`.

Reply:
Done in [c91b392](https://github.com/pyodide/pyodide/pull/4587/commits/c91b392bd7bd7dff0b5dab929d49a6ccc1480e68)